core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -218,5 +200,13 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
 }
 
 case class GoogleWorkspaceInfo(bucketName: String, policyGroupsByAccessLevel: Map[WorkspaceAccessLevel, WorkbenchEmail])
-case class ProjectTemplate(policies: Map[String, Seq[String]], services: Seq[String])
-final case class HammCromwellMetadata(bucketName: GcsBucketName, topicName: ProjectTopicName)
\ No newline at end of file
+case class ProjectTemplate(policies: Map[String, Seq[String]])
```

Comment:
why is this still a map of policies?

Reply:
DM still wants a list of owners and a list of editors, and I figured keeping them in conf would be better than putting them in the DM template directly. If you like, I can make this `case class ProjectTemplate(owners: Seq[String], editors: Seq[Strong])`

Reply:
yes please... I agree we need it in config but a map is too free form given DM names those parameters